src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
It's named for its intended use, but also because of its dependence on USE_POLLY and as a way for the user to keep his debug args and the polly args separate.
Furthermore, it can't be the expectable behaviour for an optimisation mechanism to activate a debugging mechanism.
The global nature of llvm::cl does not allow a more directed behaviour, regrettably.